torch.mul generates an incorrect result with a specific tensor shape ([2880], [1, 11, 2880])

### What happened?

The attached exception occurs when running a test case for torch.mul

### What did you expect to happen?

Generate the same result w/ marginal error

### How can we reproduce it?

Run this test case derived from gpt-oss
If the tensor shape for `var1` is `[1, 1, 2880]`, it works correctly.

```
import torch
import torch.nn as nn
import random
import operator


def get_op_callable():
    return operator.__mul__

class OpModule(nn.Module):
    def __init__(self):
        super().__init__()
        op=get_op_callable()
        self.op = op
    def forward(self, *args, **kwargs):
        return self.op(*args, **kwargs)

def SimpleIterate(device=None):
    seed = 123
    torch.manual_seed(seed)
    net = OpModule().to(device)
    backend = "inductor"
    net_compile = net if device == "cpu" else torch.compile(net, backend=backend)
    var0 = torch.rand(torch.Size([2880]), dtype=torch.float16).to(device)
    var1 = torch.rand(torch.Size([1, 11, 2880]), dtype=torch.float16).to(device)
    inputs = [var0, var1]
    kwmap = {}
    output = net_compile(*inputs, **kwmap)
    return output

if __name__ == "__main__":
    out_cpu = SimpleIterate(device="cpu")
    out_spyre = SimpleIterate(device="spyre").to("cpu")

    atol=1e-2
    rtol=1e-2
    torch.testing.assert_close(out_spyre, out_cpu, atol=atol, rtol=rtol)
```

### Any environmental details we need to know?

https://github.com/torch-spyre/torch-spyre/commit/cb8ecde29439df2230dce9cc9f0546401effd10d

### Anything else we need to know?

https://github.com/torch-spyre/torch-spyre/issues/283

### Relevant log output

```Shell
Overriding a previously registered kernel for the same operator and the same dispatch key
  operator: aten::view(Tensor(a) self, SymInt[] size) -> Tensor(a)
    registered at /pytorch/build/aten/src/ATen/RegisterSchema.cpp:6
  dispatch key: PrivateUse1
  previous kernel: registered at /pytorch/aten/src/ATen/LegacyBatchingRegistrations.cpp:1079
       new kernel: registered at /home/ishizaki/aiu-src/dt-opfunc-verify/torch-spyre/codegen/outputs/spyre_torch_ops.cpp:6213 (function operator())
Generating /tmp/torchinductor_ishizaki/inductor-spyre/sdsc_fused_mul_0__cvjvt3j/execute/sdsc_fused_mul_0/sdsc.json
RUN: sdsc_fused_mul_0 /tmp/torchinductor_ishizaki/inductor-spyre/sdsc_fused_mul_0__cvjvt3j/g2.graph.cbor
(dti-venv) ishizaki@ishizaki-spyre-opfunc-20260120-box:~/aiu-src/dt-opfunc-verify/torch-spyre/tmp$ vi test_mul_122_fp16.py
(dti-venv) ishizaki@ishizaki-spyre-opfunc-20260120-box:~/aiu-src/dt-opfunc-verify/torch-spyre/tmp$ python test_mul_122_fp16.py 
[W204 07:11:52.123278186 OperatorEntry.cpp:218] Warning: Warning only once for all operators,  other operators may also be overridden.
  Overriding a previously registered kernel for the same operator and the same dispatch key
  operator: aten::view(Tensor(a) self, SymInt[] size) -> Tensor(a)
    registered at /pytorch/build/aten/src/ATen/RegisterSchema.cpp:6
  dispatch key: PrivateUse1
  previous kernel: registered at /pytorch/aten/src/ATen/LegacyBatchingRegistrations.cpp:1079
       new kernel: registered at /home/ishizaki/aiu-src/dt-opfunc-verify/torch-spyre/codegen/outputs/spyre_torch_ops.cpp:6213 (function operator())
Generating /tmp/torchinductor_ishizaki/inductor-spyre/sdsc_fused_mul_0_lxphrf67/execute/sdsc_fused_mul_0/sdsc.json
RUN: sdsc_fused_mul_0 /tmp/torchinductor_ishizaki/inductor-spyre/sdsc_fused_mul_0_lxphrf67/g2.graph.cbor
Traceback (most recent call last):
  File "/home/ishizaki/aiu-src/dt-opfunc-verify/torch-spyre/tmp/test_mul_122_fp16.py", line 37, in <module>
    torch.testing.assert_close(out_spyre, out_cpu, atol=atol, rtol=rtol)
  File "/home/ishizaki/aiu-src/dt-opfunc-verify/dti-venv/lib64/python3.12/site-packages/torch/testing/_comparison.py", line 1589, in assert_close
    raise error_metas[0].to_error(msg)
AssertionError: Tensor-likes are not close!

Mismatched elements: 27615 / 31680 (87.2%)
Greatest absolute difference: 0.95068359375 at index (0, 8, 1929) (up to 0.01 allowed)
Greatest relative difference: inf at index (0, 0, 2586) (up to 0.01 allowed)
```

--- a/tests/inductor/test_inductor_ops.py
+++ b/tests/inductor/test_inductor_ops.py
@@ -631,6 +631,8 @@ class TestOps(unittest.TestCase, metaclass=ParameterizedTestMeta):
                     ((67, 256),) * 2,
                     ((67, 71, 256),) * 2,
                     ((7, 12, 32, 64),) * 2,
+                    # broadcasting case
+                    ((2880,), (1, 11, 2880)),
                 ]
             ),
         },

--- a/torch_spyre/_inductor/lowering.py
+++ b/torch_spyre/_inductor/lowering.py
@@ -1413,6 +1413,7 @@ def with_int64_fallback(fn, *args, convert_output=True):
 @register_spyre_lowering(
     torch.ops.aten.add.Tensor,
     type_promotion_kind=None,
+    broadcast=True,
 )
 def lower_add(x, y, *, alpha=1):
     if alpha != 1:
@@ -1431,6 +1432,7 @@ def lower_add(x, y, *, alpha=1):
 @register_spyre_lowering(
     torch.ops.aten.mul.Tensor,
     type_promotion_kind=None,
+    broadcast=True,
 )
 def lower_mul(x, y):
     return with_int64_fallback(lowering.mul, x, y)
@@ -1439,6 +1441,7 @@ def lower_mul(x, y):
 @register_spyre_lowering(
     torch.ops.aten.sub.Tensor,
     type_promotion_kind=None,
+    broadcast=True,
 )
 def lower_sub(x, y, *, alpha=1):
     if alpha != 1:
@@ -1457,6 +1460,7 @@ def lower_sub(x, y, *, alpha=1):
 @register_spyre_lowering(
     torch.ops.aten.minimum.default,
     type_promotion_kind=None,
+    broadcast=True,
 )
 def lower_minimum(x, y):
     return with_int64_fallback(lowering.minimum, x, y)
@@ -1465,6 +1469,7 @@ def lower_minimum(x, y):
 @register_spyre_lowering(
     torch.ops.aten.maximum.default,
     type_promotion_kind=None,
+    broadcast=True,
 )
 def lower_maximum(x, y):
     return with_int64_fallback(lowering.maximum, x, y)

